.travis.yml

```
@@ -4,6 +4,8 @@ matrix:
   include:
     - os: linux
       dist: bionic
+      arch:
+         - ppc64le
```

This is awesome. Could we keep on building on x86_64 at the same time?

```suggestion
      arch:
         - amd64
         - ppc64le
```